js/front/dataset/resource-modal.vue

```diff
@@ -60,3 +60,12 @@ export default {
     }
 };
 </script>
+
+<style lang="less">
+.resource-modal {
+    .dl-wide dd {
```

Comment:
Why is it duplicated from `dataset/resource/modal.vue`?

Reply:
Because they are 2 differents modals with the same issue.
They both are imported into different places (one in the front dataset page, the other in the admin dataset page) and so style isn't shared :/